pkgs/development/python-modules/torch/default.nix

```diff
@@ -95,8 +95,7 @@ let
 
     paths = with rocmPackages; [
       rocm-core clr rccl miopen miopengemm rocrand rocblas
-      rocsparse hipsparse rocthrust rocprim hipcub
-      roctracer # Unfree at the moment due to hsa-amd-aqlprofile hard dependency in rocprofiler
+      rocsparse hipsparse rocthrust rocprim hipcub roctracer
```

Comment:
This is a minor cosmetic change since the comment is outdated now.
There should be no changed to the torch package.